starwars/src/components/CharContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import CharCard from './CharCard';
+import axios from 'axios';
+
+
+const Container = styled.div`
+    border:2px solid black;
+    width: 80%;
+    margin: 0 auto;
+    display:flex;
+    flex-wrap:wrap;
+    justify-content:center;
+    justify-content: space-evenly;
+    padding:3%;
+    background: rgba(0, 0, 0, 0.5);
+    margin-bottom: 5%;
+`;
+
+
+
+const CharContainer = () => {
+
+    const [starChars, setStarChars] = useState([]);
+
+
+    useEffect(() => {
+        axios.get('https://swapi.co/api/people/')
+        .then((response) => {
+        // console.log(response);
+        console.log(response.data.results);
+        setStarChars(response.data.results);
+        })
+        .catch(error => {
+        console.log('Error: ', error);
+        })
+    },[])
+
+    // console.log(starChars);
+
+    return(
+        <Container>
+        {starChars.map(character => {
+            return(
+                <CharCard 
+                key={character.name}
```

Comment:
typically the key is an integer so you would need to pass the key / index in the map arguments and then key would be key={key} or key={index}

Reply:
I'll do this next time. I wasn't sure how to handle key in this instance and kept getting "React requires a key..." error when I tried to set it to index, so I went with name and never came back around to troubleshoot it.